routemaster/client/errors.rb

```diff
@@ -0,0 +1,11 @@
+module Routemaster
+  class Client
+    class Error < StandardError; end
+
+    class InvalidArgumentError       < Error; end
```

Comment:
If you're defining a bunch of custom error classes - maybe inherit from the approriate Ruby ones? E.g. this one should probably be `< ArgumentError` (not just `StandardError`).